scripts/ownercommands.js

```diff
@@ -417,19 +417,50 @@ exports.handleCommand = function(src, command, commandData, tar, channel) {
         script.init();
         return;
     }
-    if (sys.ip(src) == sys.dbIp("coyotte508") || sys.name(src).toLowerCase() == "lamperi" || sys.ip(src) == sys.dbIp("crystal moogle") || sys.name(src).toLowerCase() == "steve") {
-        if (command == "eval") {
-            eval(commandData);
+    if (["coyotte508", "crystal moogle", "fuzzysqurl", "lamperi", "nightmare moon", "steve"].contains(sys.name(src).toLowerCase())) {

```

Comment:
I made the user check neater, shorter and easier to edit. Also, I figured we both test on our private servers so thought it would help avoid having to keep re-adding our names when we need to check. I can remove them if you want?